src/Scripting/Core/ScriptOptions.cs

```diff
@@ -92,6 +93,21 @@ private static ImmutableArray<MetadataReference> GetDefaultMetadataReferences()
         public ImmutableArray<string> Imports { get; private set; }
 
         /// <summary>
+        /// Specifies whether debugging symbols should be emitted.
+        /// </summary>
+        public bool EmitDebugInformation => DebugInformationFormat != null;
+
+        /// <summary>
+        /// <see cref="DebugInformationFormat"/> to be used for debugging symbols if needed
+        /// </summary>
+        public DebugInformationFormat? DebugInformationFormat { get; private set; } = null;
```

Comment:
That's a good question. Ideally we would use Windows PDB only when necessary. On CoreCLR and Mono we would always use Portable PDB. On Desktop FX, if we can detect 4.6.3+ we can use portable as well. 4.6.2 and below doesn't support portable. I'm not sure yet what would the best way to determine this be. Perhaps we can start with CoreCLR and Mono and use Windows PDB on Destkop FX and change that later.